locale/fr.po

```diff
@@ -0,0 +1,70 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2017-06-26 12:18+0200\n"
+"PO-Revision-Date: 2017-06-26 12:22+0200\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: fr_FR\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 2.0.2\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#: init.lua:19
+msgid "Steel blacksmithing hammer"
+msgstr "Marteau de forgeron en acier"
+
+#: init.lua:20
+msgid "A tool for repairing other tools at a blacksmith's anvil."
+msgstr "Un outil pour réparer les autres outils avec une enclume de forgeron."
+
+#: init.lua:21
+msgid ""
+"Use this hammer to strike blows upon an anvil bearing a damaged tool and you "
+"can repair it. It can also be used for smashing stone, but it is not well "
+"suited to this task."
+msgstr ""
+"Utilisez ce marteau pour frapper une enclume contenant un outil endommage, "
+"ainsi vous pourrez le réparer. Il peut etre aussi utiliser pour casser de la "
+"pierre, mais il n'est pas adapté à cette tâche."
+
+#: init.lua:98
+msgid "Anvil"
+msgstr "Enclume"
+
+#: init.lua:99
+msgid ""
+"A tool for repairing other tools in conjunction with a blacksmith's hammer."
+msgstr ""
+"Un outil pour réparer les autres outils à utiliser avec un marteau de "
+"forgeron."
+
+#: init.lua:100
+msgid ""
+"Right-click on this anvil with a damaged tool to place the damaged tool upon "
+"it. You can then repair the damaged tool by striking it with a blacksmith's "
+"hammer. Repeated blows may be necessary to fully repair a badly worn tool. "
+"To retrieve the tool either punch or right-click the anvil with an empty "
+"hand."
+msgstr ""
+"Cliquez-droit sur cette enclume avec un outil endommagé pour le placer "
+"dessus. Vous pourrez alors réparer l'outil endommagé en le frappant avec un "
+"marteau de forgeron. Des coups successifs seront nécessaire pour réparer "
+"l'outil entiérement. Pour récupérer l'outils, frappez dessus ou faites un "
```

Comment:
l'outil

Reply:
Do you mean I should remove the "s" from the end of l'outils?

Apologies, I'm a terrible Canadian and don't actually know much French so I want to be completely sure I'm changing the right things here. :)

Reply:
Yes, exactly that.

Reply:
I'll made the fixes as soon as possible thnx for the replies.